DirectEmployers/MyJobs
# [PD-2756] modify dot jobs logo to match prod

Purpose: Modify current svg dot jobs logo to match Prod png logo, except for the blue dot.

To Test:

1. Please switch to v2 template.

2. Please ensure that the v2 dot jobs logo matches the Production logo except for the blue dot.  Please also ensure there aren't any phallus symbols accidentally checked in, this will disappoint Jon Koons, but hey, I value my job.
